examples/morphology/main.go

```diff
@@ -0,0 +1,35 @@
+// Port of http://members.shaw.ca/el.supremo/MagickWand/grayscale.htm to Go
+package main
+
+/*
+#cgo CFLAGS: -I/usr/local/include/ImageMagick-6
+#cgo LDFLAGS: -L/usr/sbin/lib
```

Comment:
I posted a more detailed answer in your SO question. But for completeness here, you should check out the README for the project. It says how to use custom ImageMagick locations using the build tag: https://github.com/gographics/imagick#build-tags